apps/communications/contacts/test/unit/views/form/boot_test.js

```diff
@@ -0,0 +1,175 @@
+/* global LazyLoader, Form, FormUI */
+
+'use strict';
+
+require('/shared/test/unit/load_body_html_helper.js');
+require('/shared/js/lazy_loader.js');
+requireApp('communications/contacts/views/form/js/boot.js');
+
+
+suite('Boot', function() {
```

Comment:
nit: despite that this is related to the form, we could name the suite 'Form boot', to identify it easily on the logs